automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -221,15 +221,69 @@ class MethodConfigTabSpec extends FreeSpec with WebBrowserSpec with CleanUp with
 
   "delete a method config from a workspace" in withWebDriver { implicit driver =>
     withWorkspace(billingProject, "TestSpec_FireCloud_delete_method_from_workspace") { workspaceName =>
-      api.methodConfigurations.copyMethodConfigFromMethodRepo(billingProject, workspaceName, TestData.SimpleMethodConfig.namespace,
-        TestData.SimpleMethodConfig.name, TestData.SimpleMethodConfig.snapshotId, TestData.SimpleMethodConfig.namespace, methodConfigName)
+      api.methodConfigurations.copyMethodConfigFromMethodRepo(billingProject, workspaceName, MethodData.SimpleMethodConfig.configNamespace,
+        MethodData.SimpleMethodConfig.configName, MethodData.SimpleMethodConfig.snapshotId, MethodData.SimpleMethodConfig.configNamespace, methodConfigName)
 
       signIn(uiUser)
-      val methodConfigDetailsPage = new MethodConfigDetailsPage(billingProject, workspaceName, TestData.SimpleMethodConfig.namespace, methodConfigName).open
+      val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProject, workspaceName, MethodData.SimpleMethodConfig.configNamespace, methodConfigName).open
       val workspaceMethodConfigPage = methodConfigDetailsPage.deleteMethodConfig()
 
       workspaceMethodConfigPage.filter(methodConfigName)
-      assert(find(methodConfigName).size == 0)
+      assert(find(methodConfigName).isEmpty)
+    }
+  }
+
+  "For a method config that references a redacted method" - {
+    "should be able to choose new method snapshot" in withWebDriver { implicit driver =>
+      withWorkspace(billingProject, "MethodConfigTabSpec_redacted_choose_new_snapshot") { workspaceName =>
+        withConfigForRedactedMethodInWorkspace("MethodConfigTabSpec_redacted_choose_new_snapshot", billingProject, workspaceName, true) { configName =>
+          signIn(uiUser)
+          val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProject, workspaceName, MethodData.SimpleMethodConfig.configNamespace, configName).open
+          assert(methodConfigDetailsPage.ui.isSnapshotRedacted())
+
+          methodConfigDetailsPage.ui.openEditMode()
+          methodConfigDetailsPage.ui.saveEdits("disabled")
```

Comment:
I think it would read a little better as something like:
```
    methodConfigDetailsPage.ui.checkSaveButtonState shouldEqual "disabled"
    methodConfigDetailsPage.ui.clickSaveButton()
```

Reply:
+1